File: test/fixtures/javascript/findReferences/consumer.js
// Consumer file that uses symbols from main.js and utils.js
const { calculateSum, createUser } = require('./main');
const { processNumbers, LOCAL_CONSTANT } = require('./utils');

class DataProcessor {
  calculate() {
    return calculateSum(5, 15); // Reference to calculateSum from main.js
  }
  
  processUser() {
    const user = createUser(42, 'Consumer User'); // Reference to createUser from main.js
    const result = processNumbers(3, 4);
    return {
      ...user,
      name: `User ${LOCAL_CONSTANT}` // Reference to LOCAL_CONSTANT from utils.js
    };
  }
}

// Test case for no references found
function unusedFunction() {
  return 'This function should have no references';
}

module.exports = { DataProcessor, unusedFunction };

File: test/fixtures/python/find_references/consumer.py
# Consumer file that uses symbols from main.py and utils.py
from .main import calculate_sum, UserData
from .utils import process_numbers, LOCAL_CONSTANT


class DataProcessor:
    def calculate(self) -> int:
        return calculate_sum(5, 15)  # Reference to calculate_sum from main.py
    
    def process_user(self) -> UserData:
        user = UserData(42, 'Consumer User')  # Reference to UserData from main.py
        result = process_numbers(3, 4)
        user.name = f'User {LOCAL_CONSTANT}'  # Reference to LOCAL_CONSTANT from utils.py
        return user


# Test case for no references found
def unused_function():
    return 'This function should have no references'